SR2019/2019-07-02-kit-chase-round-2.md

```diff
@@ -0,0 +1,30 @@
+---
+to: Teams with still outstanding kit
+subject: Outstanding kit - Round 2
+---
+
+Hello!
+
+We sent an email to you back in May stating we required a kit back from you. Unfortunately, we've not heard a response.
+
+The kits are loaned to you fur the duration of the competition year, and as this has ended, we need the kits back, to prepare for future events.
+
+Please securely pack the kit and ship it to the address listed on [our website](https://studentrobotics.org/contact/). Once you have dispatched it, please [let us know](mailto:teams@studentrobotics.org).
+
+## How to ship your kit
+
+- Kits should be shipped in the white 18l Really Useful Box (RUB). For extra protection, we recommend wrapping this in paper / cling film during shipping.
+- All parts of the kit should be suitably protected either in Jiffy bags or bubble wrap.
+- Damaged batteries must not be shipped. If you have concerns about your batteries, please contact us.
+- Any empty space in the box should be filled with paper or bubble wrap. Please don't use packing peanuts as they're messy.
+
+Please insure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.
```

Comment:
```suggestion
Please ensure the kit during shipping and use a courier which supports tracking, to ensure the kit isn't lost.
```

Reply:
Maybe also mention if the kit is lost they're expected to repay us? This is the mean email anyway.

Reply:
Regarding the first comment in this chain - I'm pretty sure the original text is correct.:
- "insure" is the verb form of obtaining insurance, which is the intent here
- "ensure" means "make happen"  (note the later usage of this in the same sentence, which is also correct)

Reply:
"Insure" is correct here.

I think i'm ok with leaving out the fact they have to repay us for it. This isn't actually the mean email, the properly mean one is when we start contacting people higher up in the school.